services-custom/dynamodb-enhanced/src/main/java/software/amazon/awssdk/enhanced/dynamodb/internal/extensions/DefaultDynamoDbExtensionContext.java

```diff
@@ -96,6 +107,7 @@ public int hashCode() {
         result = 31 * result + (operationContext != null ? operationContext.hashCode() : 0);
         result = 31 * result + (tableMetadata != null ? tableMetadata.hashCode() : 0);
         result = 31 * result + (tableSchema != null ? tableSchema.hashCode() : 0);
+        result = 31 * result + (operationType != null ? operationType.hashCode() : 0);
```

Comment:
Do we have equals hashcode test? If not, can we add one using equalsVerifier?

Reply:
good suggestion!